integration_tests/mockData/actionPlanByPrisonNumberData.ts

```diff
@@ -0,0 +1,199 @@
+const actionPlans = {
+  A00001A: {
+    request: {
+      method: 'GET',
+      urlPattern: '/action-plans/A00001A',
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: {
+        prisonerNumber: 'A00001A',
+        goals: [
+          {
+            goalReference: '10efc562-be8f-4675-9283-9ede0c19dade',
+            title: 'Learn first aid',
+            status: 'ACTIVE',
+            steps: [
+              {
+                stepReference: '177e45eb-c8fe-438b-aa81-1bf9157efa05',
+                title: 'Book first aid course',
+                targetDateRange: 'THREE_TO_SIX_MONTHS',
+                status: 'NOT_STARTED',
+                sequenceNumber: 1,
+              },
+              {
+                stepReference: '32992dd1-7dc6-4480-b2fc-61bc36a6a775',
+                title: 'Complete first aid course',
+                targetDateRange: 'SIX_TO_TWELVE_MONTHS',
+                status: 'NOT_STARTED',
+                sequenceNumber: 2,
+              },
+            ],
+            createdBy: 'auser_gen',
+            createdAt: '2023-07-20T09:29:15.386Z',
+            updatedBy: 'auser_gen',
+            updatedAt: '2023-07-20T09:29:15.386Z',
+            reviewDate: null,
+            notes: "Pay close attention to Paris' behaviour.",
```

Comment:
This seems to be *note* singular in the template - need to double check

Reply:
That's right, it's `notes` in the API, but `note` in the UI project.